sunscreen_compiler_common/src/context.rs

```diff
@@ -341,7 +341,11 @@ where
     /**
      * Represents the program context. Tracks groups of nodes in the compilation graph.
      */
-    pub group_stack: Vec<Group>,
+    pub group_stack: Vec<u64>,
```

Comment:
I expect a stack of `Group`s here. When you call add_node, you pass the entire stack to the group trie and lookup the group_id.

Reply:
Actually, I think we can keep this as is.